README.md

```diff
@@ -7,58 +7,41 @@ Elasticsearch configuration using [microcosm](https://github.com/globality-corp/
 
 ## Usage
 
-    from microcosm.api import create_object_graph
-
-    graph = create_object_graph(name="foo")
-    graph.use("elasticsearch_client")
-
-    # create an index
-    graph.elasticsearch_client.indices.create(
-        index="my-index",
-        body=dict(
-            settings=dict(
-                "number_of_shards": 1,
-                "number_of_replicas": 0,
-            )
-        )
-    )
-
-    # bulk index documents
-    from elasticsearch.helpers import streaming_bulk
-    docs_generator = some_generator_expression_yielding_dicts
-    for ok, result in streaming_bulk(
-        graph.elasticsearch_client,
-        docs_generator,
-        index="my-index",
-        doc_type="my-doc-type",
-        chunk_size=100,
-        refresh=True,
-        ):
-        if not ok:
-            raise RuntimeError("Bulk indexing failed for batch!")
-
-
-## Convention
-
- - Support for signed requests for AWS Elasticsearch using AWS4Auth
- - Support for IAM Role-based credentials based on instance metadata when running on EC2
- - Proper unicode support for Python 2.x / 3.x codebases
+ - Provides a `microcosm` compatible `Elasticsearch` client
+    -  Supports signed requests for AWS Elasticsearch using AWS4Auth
+    -  Supports IAM Role-based credentials based on instance metadata when running on EC2
+    -  Provides proper unicode support for Python 2.x / 3.x codebases
+
+ - Includes an implementation of a persistence `Store` using Elasticsarch
+
+
+## Testing
+
+Unit tests depend on a running instance of Elasticsearch:
+
+ 1. Bring up the ES with `docker-compose`:
+
+         docker-compose up -d
+
+ 2. Run tests:
+
+         nosetests
 
 
 ## Configuration
 
 When using with an AWS Elasticsearch instance, use:
 
-    config.elasticsearch_client.use_aws4auth = True
+    config.elasticsearch_client.use_aws4auth = 'true'
```

Comment:
Configuration parameters now assume strings to support environment variables properly.